src/components/navBar.js

```diff
@@ -0,0 +1,38 @@
+import { NavLink } from 'react-router-dom';
+
+function NavBar() {
+  return (
+    <nav>
+      <div>
+        <NavLink className="logo" to="/math-magicians">
+          Math Magicians
+        </NavLink>
```

Comment:
Same here Please make the link to the home page `to="/"` instead of `to="/math-magicians"`.